Docker_Container/README.md

```diff
@@ -0,0 +1,54 @@
+## Steps to start the build with Docker 
```

Comment:
@yasharth291  

1. The `README.md` file in the root directory doesn't have a link to an installation guide. No one will know how to install `talawa-admin` based on current documentation.
1. Not everyone will want to use docker, so I suggest moving the `Docker_Container/README.md ` to the root directory as `INSTALLATION.md`, create a section for regular installation and another section with the contents of the `Docker_Container/README.md ` file.

Reply:
okay sir changing them